### PR TITLE
KOGITO-1830: BDD tests: Test service discovery on Spring Boot applica…

### PR DESCRIPTION
**Pull request by Sgitario**
…tion

JIRA Ticket: https://issues.redhat.com/browse/KOGITO-1830
Description: Cover service discovery test for Spring Boot.

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: test/features/service_discovery.feature [hunk 1]
```diff
@@ -54,4 +54,51 @@ Feature: Discovery with onboarding
     @native
     Examples: Native
       | native  | minutes |
-      | enabled | 20      |
\ No newline at end of file
+      | enabled | 20      |
+
+#####
+
+  Scenario Outline: Deploy Spring Boot onboarding example
```
Comment (sutaakar) [inline test/features/service_discovery.feature:61]: This can be transformed to scenario as there are no parameter combinations to be executed.

Comment (Sgitario) [reply test/features/service_discovery.feature]: Changed and tested again in Openshift.

**Comment by sutaakar** [comment]
Should be merged together with https://github.com/kiegroup/kogito-examples/pull/226

**Comment by ricardozanini** [comment]
Has one conflict after the merges.

**Comment by Sgitario** [comment]
The conflict has been sorted it out. I checked the tests within these changes again and they worked fine.

**Comment by radtriste** [comment]
next time we merge too early a PR because of a dependent PR no merged yet, we should revert it, this would avoid to have problems in our tests ... :(
anyway, no matter here anymore, https://github.com/kiegroup/kogito-examples/pull/226 should be merged tomorrow or Thursday (all green)

**Pull request merged**